tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -470,7 +464,7 @@ def test_backup_and_restore_exclude_from_pattern(ansible_zos_module):
         )
         results = hosts.all.zos_backup_restore(
             operation="backup",
-            data_sets=dict(include=DATA_SET_PATTERN, exclude=DATA_SET_NAME2),
+            data_sets=dict(include=DATA_SET_NAME, exclude=DATA_SET_NAME2),
```

Comment:
This change also affects the test case, that is because in this part we are looking to match a pattern. Fix should be simple, instead of using the complete data set name just strip from right until the next dot and then add "*", that should be your pattern to test.